.gitlab-ci.yml

```diff
@@ -54,6 +59,11 @@ lint_package_build_script:
   script:
     - shellcheck -C -f tty -s sh build.sh
 
+lint_prepare_disk_script:
+  <<: *shell_linting_common
+  script:
+    - shellcheck -C -f tty -s sh scripts/30_prepare_disk.sh
```

Comment:
General comment, wouldn't it make more sense to just "glob" the shell files, and thus have the CI always check all shell files, instead of needing to maintain a "shadow list"?

Reply:
Yes good point, the shadow list is a maintenance burden. I will see if this works.